server_environment_ir_config_parameter/tests/config_param_test.xml

```diff
@@ -0,0 +1,6 @@
+<odoo>
+    <record model="ir.config_parameter" id="some_record_id">
```

Comment:
Should be something else, no? `some_record_id`

Reply:
it's just for tests - the xml_id is not important. what matters is the key and value